examples/showcase-curved-mockups/index.html

```diff
@@ -8,18 +8,19 @@
   </head>
   <body>
     <a-scene>
+      <a-curvedimage src="../_images/ui/3.png" radius="5.7" theta-length="180" height="9"
+                     rotation="0 90 0" scale="1.2 1.2 1.2"></a-curvedimage>
+      <a-curvedimage src="../_images/mozvr.png" radius="5.7" theta-length="17" height=".36"
+                     opacity="0.2" rotation="0 250 0" position="0 2 0"></a-curvedimage>
+      <a-curvedimage src="../_images/ui/1.png" radius="5.7" theta-length="72" height="2.6"
+                     rotation="0 80 0" position="0 0.7 0" scale="0.6 0.6 0.6"></a-curvedimage>
+      <a-curvedimage src="../_images/ui/2.png" radius="5.7" theta-length="70" height="3.02"
+                     rotation="0 -130 0" scale="0.8 0.8 0.8"></a-curvedimage>
 
-      <a-camera fov="80" position="0 1.8 0" near="0.2" cursor-color="#33CCFF"></a-camera>
```

Comment:
[http://localhost:9000/examples/showcase-curved-mockups/](http://localhost:9000/examples/showcase-curved-mockups/)

camera regressed and cursor missing:

<img width="1363" alt="screenshot 2016-02-17 03 18 01" src="https://cloud.githubusercontent.com/assets/203725/13107970/313af2b0-d525-11e5-9fa8-c4a1d880e053.png">

<img width="1363" alt="screenshot 2016-02-17 03 08 28" src="https://cloud.githubusercontent.com/assets/203725/13107969/313aa990-d525-11e5-8782-43fc459cc24e.png">